docs/tutorials/getting_started.ipynb

```diff
@@ -0,0 +1,287 @@
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "1. Load your data into a `Table`, the data is available under `docs/tutorials/data/titanic.csv`:\n"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "outputs": [],
+   "source": [
+    "from safeds.data.tabular.containers import Table\n",
+    "\n",
+    "titanic = Table.from_csv_file(\"data/titanic.csv\")"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "start_time": "2023-06-20T13:51:49.792239Z",
+     "end_time": "2023-06-20T13:51:51.528683Z"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "2. Split the titanic dataset into two tables. A training set, that we will use later to implement a training model to predict the survival of passengers, containing 60% of the data, and a testing set containing the rest of the data.\n",
+    "Delete the column `survived` from the test set, to be able to predict it later:"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "split_tuple = titanic.split(0.60)\n",
+    "\n",
+    "train_table = split_tuple[0]\n",
+    "test_table = split_tuple[1]\n",
+    "\n",
+    "test_table = test_table.remove_columns([\"survived\"])"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "3. Use `OneHotEncoder` to create an encoder, that will be used later to transform the training table.\n",
+    "* We use `OneHotEncoder` to transform non-numerical categorical values into numerical representations with values of zero or one. In this example we will transform the values of columns sex and travel_class, hence they will be used in the model for predicting the surviving of passengers.(In this case, travel_class has categorical but not numerical values, OneHotEncoder Would still be the most efficient Encoder to use in this case, since we can pass both columns needed as features at the same time)\n",
+    "* Use the `fit` function of the `OneHotEncoder` to pass the table and the column names, that will be used as features to predict who will survive to the encoder.\n",
+    "* The names of the columns before transformation need to be saved, because `OneHotEncoder` will change the names of the fitted `Column`s:\n"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "C:\\Users\\hussi\\AppData\\Local\\Temp\\ipykernel_16796\\1962289518.py:4: UserWarning: The columns ['travel_class'] contain numerical data. The OneHotEncoder is designed to encode non-numerical values into numerical values\n",
+      "  encoder = OneHotEncoder().fit(train_table, [\"sex\", \"travel_class\"])\n"
```

Comment:
```suggestion
      "  encoder = OneHotEncoder().fit(train_table, [\"sex\"])\n"
```

Don't apply the OneHotEncoder on the `travelclass` column as it already contains numerical values and a warning is thrown when applied.